### PR TITLE
RE : add thai locale and not spacer between id position

### PR DESCRIPTION
**Pull request by ghost**
sorry to make a bug. 😭

### CODE REVIEW

File: lib/src/identifier_position.dart [hunk 1]
```diff
@@ -1 +1 @@
-enum IdentifierPosition { prepend, append, dontDisplay }
+enum IdentifierPosition { prepend, prependUnSpace, append , appendUnSpace, dontDisplay }
```
Comment (rinukkusu) [inline lib/src/identifier_position.dart]: Hi, thanks for helping out! Can you please rename the enum values to a more english sounding name, though?
Maybe something like `prependWithoutSpace` and `appendWithoutSpace`?

**Review by rinukkusu — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
Change enum name un to without

File: lib/src/identifier_position.dart (hunk 1 above)
Comment (ghost) [reply lib/src/identifier_position.dart]: Thank, i'm changed it. 😇
Sorry, for my language. 😅

**Review by ghost — COMMENTED** [review]
Fixed LocaleTh on pastPosition

**Pull request merged**